src/libopensc/card-cac.c

```diff
@@ -1499,7 +1499,7 @@ static struct sc_card_operations cac_ops;
 
 static struct sc_card_driver cac_drv = {
 	"Common Access Card (CAC)",
-	"cac",
+	"CAC",
```

Comment:
I'm sorry to be pedantic about this. Would it be possible to change the shortname to lowercase? The PIV's short name is only uppercase, because we need to cope with the mistakes of the past.